addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -1528,7 +1536,7 @@ body.editor_enable.editor_has_snippets {
                 margin-top: 0;
                 min-width: 4em; // Ideally rely on actual natural min-width, but does not work...
             }
-            we-button, .o_we_so_color_palette {
+            we-button, .o_we_so_color_palette, we-button-group {
```

Comment:
Actually this change is in the wrong commit, it should have been in the "scroll animation" commit..
It is for the buttons "out" and "in" that appears next to the animation option selector..  
If we want this in stable, it would be for custos but for our code we don't need it

Reply:
Ok, I would make this kind of change in stable but maybe this is a waste of time. You can add it in the right commit, we can backport if somehow actually complains (probably never).